### ISSUE TITLE
ImportError due to changes in 3.4.3

### ISSUE THREAD

**Issue by craigminihan** [allrod5/injectable#41]
On an existing codebase I am seeing `ImportError`s when upgrading from 3.4.1 to latest.

The error I am seeing is: `ImportError: attempted relative import with no known parent package`

Prior to 3.4.3 all modules load successfully.

I've not had time to put an example together showing the exact error case yet. I'll follow up with this later.

Stack:
```text
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/craig/code/XXX.py", line 7, in <module>
    injectable.load_injection_container(search_path=os.getenv("PYTHONPATH"))
  File "/home/craig/.local/lib/python3.6/site-packages/injectable/container/load_injection_container.py", line 43, in load_injection_container
    search_path, default_namespace or DEFAULT_NAMESPACE
  File "/home/craig/.local/lib/python3.6/site-packages/injectable/container/injection_container.py", line 158, in load_dependencies_from
    run_path(file.path)
  File "/usr/lib/python3.6/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/usr/lib/python3.6/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "MYFILE", line 3, in <module>
    from .MYCLASS
ImportError: attempted relative import with no known parent package
```

**Comment by allrod5** [comment]
Thanks for the issue @craigminihan! I haven't got the time to reproduce it yet though I suspect [this change released in 3.4.3](https://github.com/allrod5/injectable/pull/31/files#diff-6f2af16c51d484519a4fe848e6c9f9f1R158) is the source of the problem. I'll dig into it in the coming week most probably and I'll keep you posted!

**Comment by allrod5** [comment]
Just to follow up on this issue: I haven't got the time yet to reproduce this issue and further investigate. It's the next thing in the backlog though I cannot guarantee an exact time schedule to look into this. You can expect though for me to come back in no more than 4 weeks from now.

Thank you for the effort put in reporting the issue, it is very much appreciated. If anyone is willing to look into this, feel free to do it, just mention it here in the comments so we don't get multiple people working on the same issue.

**Comment by allrod5** [comment]
Hi @craigminihan, just passing by to inform you that the fix to the reported issue has been released in version 3.4.7 and is already available at PyPI. Thanks for your help on this bug! :tada:
